.travis.yml

```diff
@@ -18,7 +18,10 @@ matrix:
   include:
     - php: 5.5
     - php: 7.0
-      env: 
+      env:
+        - SYMFONY__DATABASE__DRIVER=pdo_pgsql
+        - SYMFONY__DATABASE__USER=postgres
+        - SYMFONY__DATABASE__PASSWORD=postgres
```

Comment:
No, only on travis, but AFAIK postgres reuses the users from the system, and I bet not too many people have a user called postgres then. So they have to pass these environment variables anyway. Apart from that we also don't do that for switching between the two different jackalope implementations.